docs/man/docker-cp.1.md

```diff
@@ -2,18 +2,55 @@
 % Docker Community
 % JUNE 2014
 # NAME
-docker-cp - Copy files/folders from the PATH to the HOSTPATH, or STDOUT
+docker-cp - Copy files or folders from a container's PATH to a HOSTDIR
+on the host where the command is run.
 
 # SYNOPSIS
 **docker cp**
 [**--help**]
-CONTAINER:PATH HOSTPATH|-
+CONTAINER:PATH HOSTDIR|-
 
 # DESCRIPTION
-Copy files/folders from a container's filesystem to the
-path. Use '-' to write the data as a tar file to STDOUT.
-Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+
+Copy files or folders from a `CONTAINER:PATH` to the `HOSTDIR`.  The
+`CONTAINER:PATH` is relative to the root of the container's filesystem. You can
+copy from a running or stopped container. The `HOSTDIR` refers to a directory
+on the host. 
+
+The `PATH` can be a file or directory. The `docker cp` command assumes all paths
+start at the `/` (root) directory. This means supplying the initial forward
+slash is optional; The command sees `compassionate_darwin:/tmp/foo/myfile.txt`
+and `compassionate_darwin:tmp/foo/myfile.txt` as identical.
+
+If you do not specify an absolute path for your `HOSTDIR` value, Docker creates
+the directory relative to where you run the `docker cp` command. For example, suppose you
+want to copy the `/tmp/foo` directory from a container to the `/tmp` directory
+on your host. If you run `docker cp` in your `~` (home) directory on the host:
+
+		$ docker cp compassionate_darwin:tmp/foo /tmp
+
+Docker creates a `/tmp/foo` directory on your host. Alternatively, you can omit
+the leading slash in the command. If you execute this command from your home directory:
+
+		$ docker cp compassionate_darwin:tmp/foo tmp
+
+Docker creates a `~/tmp/foo` subdirectory in your home directory.  
+
+When copying files, the command appends to rather than overwrites the
```

Comment:
I wouldn't use the word 'append'.  My first reaction was that it would append files - meaning concatenate the data in them. I would say something more like:  When copying files, if the destination directory already exists then the source files are copied into it without impacting any other pre-existing files in that directory.